src/lib/mo-pdf-viewer.component.ts

```diff
@@ -107,624 +155,384 @@ export class MoPdfViewerComponent {
       icon: faTag,
     },
     {
-      label: 'Comments',
+      label: 'Notes',
       icon: faMessage,
     },
   ];
-  public navs = Object.values(RIGHT_PANE_NAV);
-  @Input()
-  public activeNav = this.navs[0];
-  @Output()
-  public closePane = new EventEmitter();
-  @Output()
-  public activeNavChange = new EventEmitter<string>();
+
   public selectedSearchCategory = 'Highlights';
-  public page = 1;
-  public highlightPipe = new HighlightPipe();
-  public componentRefs: ComponentRef<TextOptionsComponent>[] = [];
-  public dropdownVisible: { [key: string]: boolean } = {};
-  constructor(
-    private readonly router: Router,
-    private readonly route: ActivatedRoute,
-    private readonly viewContainerRef: ViewContainerRef,
-    public elementRef: ElementRef
-  ) {}
-
-  public afterLoadComplete(): void {
-    const queryParams = this.route.snapshot.queryParams;
-    if (queryParams['page']) {
-      this.page = queryParams['page'];
-    }
-  }
 
   public selectSearchCategory(category: string): void {
     this.selectedSearchCategory = category;
   }
 
-  public onKeyUp(event: KeyboardEvent): void {
```

Comment:
These changes seem too extensive. Why does all this need to be removed? (same on the HTML side). Changes should be minimal to avoid conflicts with updates we may need to pull in.

Reply:
These codes for used for the ng2 pdf-viewer custom search. This is not used now so the codes related to it is not required

Reply:
Even if it's not used, we shouldn't touch anything we don't have to or else pulling in updates will be much harder. Does leaving it in break anything?

Reply:
Leaving it in place may affect the code coverage. Also, the code I previously wrote is no longer in use, as we initially used ng2-pdf-viewer and are now using ngx-extended-pdf-viewer, which are completely different.